docs/tutorial.md

```diff
@@ -343,46 +352,60 @@ Some of the new styles have more than one selector separated by a space. The spa
 }
 ```
 
-The `.started` selector matches any widget with a `"started"` CSS class. While `#start` matches a child widget with an ID of `"start"`. So it matches the Start button only for Stopwatches in a started state.
+The `.started` selector matches any widget with a `"started"` CSS class. While `#start` matches a child widget with an ID of `"start"`.
+When combined with a space, the the selector will match the start button *only* if it is inside a container with a CSS class of "started".
```

Comment:
I had to re-read this 2-3 times. Might want to reiterate that "the selector" is referring to `.started #start`.

```suggestion
When combined with a space, the selector will match the start button *only* if it is inside a container with a CSS class of "started".
```